README.md

```diff
@@ -1,2 +1,241 @@
+<br>
+
+# 📖 Bookstore-React-App (bookstore-react-app) <a name="bookstore-react-app"></a>
+
+ The Bookstore is a website that allows you to display a list of books, add a book and remove a book!ics.
+<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The following stacks were used
+
+<details style="color:rgb(87, 247, 255);">
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a></li>
+    <li><a href="hhttps://create-react-app.dev/">Create React App</a></li>
+ </ul>
+</details>
+
+<!-- Features -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **ReactJS library**
+- **Using JSX syntax**
+- **Unit test with jest**
+- **Boostrap**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ### 🚀 Live Demo <a name="live-demo"></a>
+<a href=#>Live Demo</a> -->
+
+<!-- GETTING STARTED -->
+<br>
+
+# 💻 Getting Started <a name="getting-started"></a>
+
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```bash
+  git clone git@github.com:PabloBona/bookstore-react-app.git
+```
+<br>
+
+# 📖 Install
+
+Install this project's dependencies with:
+```
+cd bookstore-react-app
+npm install
+```
+
+<br>
+
+# 📖 Usage
+
+To run the project, execute the following command:
+
+```bash
+ npm run start
+```
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+<br>
+
+# Run tests
+
+```bash
+  npm test
+```
+
+<br>
+
+# 📖 Run linterns tests
+
+If you follow the tutorial above to setup linters then you can run these tests
+
+```$
+  npx hint .
+```
+
+```$
+  npx stylelint "**/*.scss"
+```
+
+or if you use css then run this instead of the latter command above
+
+```$
+  npx stylelint "**/*.{css,scss}"
+```
+
+<br>
+
+# Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
```

Comment:
- [ ] [OPTIONAL] Please consider removing the default react readme instructions. This is because the instructions are already stated above this ☺️.